lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -478,7 +478,10 @@ class MayaPathMapper : public Fvp::PathMapper
 
     Fvp::PrimSelections 
     UfePathToPrimSelections(const Ufe::Path& appPath) const override {
-        return _piSi.UfePathToPrimSelectionsLit(appPath);
+        auto litPaths = _piSi.UfePathToPrimSelectionsLit(appPath);
```

Comment:
Awkwardly, when selected, Maya objects have geometry under two Hydra prim hierarchies, with selection highlighting geometry (poly wire) rooted under `/MayaHydraViewportRenderer/rprims`, and the real mesh geometry under `/MayaHydraViewportRenderer/rprims/Lighted`.  Both must be considered "selected" for isolate select to preserve them in the scene, otherwise the unlit selection highlighting geometry will be hidden.

Reply:
I recall we have a tech debt to remove /Lighted for lit prims, something worth a look later